core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -317,6 +318,9 @@ protected void startServingRPCServer() {
           TimeUnit.MILLISECONDS);
 
       serverBuilder.executor(mRPCExecutor);
+      MetricsSystem.registerGaugeIfAbsent(MetricKey.MASTER_RPC_PENDING_JOBS.getName(),
```

Comment:
As we create mRPCExecutor as a concrete class `ForkJoinPool`, and using the `getQueuedTaskCount` method which only belong to the `ForkJoinPool`, so my suggestion is define mRPCExecutor as the ForkJoinPool.

Reply:
> As we create mRPCExecutor as a concrete class `ForkJoinPool`, and using the `getQueuedTaskCount` method which only belong to the `ForkJoinPool`, so my suggestion is define mRPCExecutor as the ForkJoinPool.

I agree with you

Reply:
Please remove the unused `import java.util.concurrent.ExecutorService;` also

Reply:
JFYI; `ForkJoinPool` might go away soon.

Reply:
> JFYI; `ForkJoinPool` might go away soon.

shall we leave a TODO or doc when defining `mRPCExecutor` ?

Reply:
@apc999 How should todo be described